Signum.React.Extensions/Chart/D3Scripts/StackedBars.tsx

```diff
@@ -123,9 +125,20 @@ export default function renderStackedBars({ data, width, height, parameters, loa
               return undefined;
 
             var key = keyColumn.getKey(r.data.rowValue);
+            var rowByKey = rowsByKey[key];
+
+            let totalCount = 0;
+            let sKeys = [] as string[];
+            stackedSeries.map(s => { sKeys.push(s.key) });
+            for (var i = 0; i < sKeys.length; i++) {
+              let v = rowByKey.values[sKeys[i]];
+              if (v && v.value) {
+                totalCount = totalCount + v.value;
+              }
+            }

```

Comment:
Two questions: 
* Is there a reason to use `rowByKey` instead of just `row` variable on the top? 
* Can not the whole loop be something like this: `const sKeys = stackedSeries.sum(s => rowByKey.values[s.key]?.value  ?? 0)`